.swcrc

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://swc.rs/schema.json",
+  "jsc": {
+    "parser": {
+      "syntax": "typescript",
+      "decorators": true,
+      "topLevelAwait": true,
+      "importMeta": true
+    },
+    "transform": {
+      "legacyDecorator": true,
+      "decoratorMetadata": true,
+      "useDefineForClassFields": true
+    },
+    "minify": {
+      "compress": {
+        "unused": true
+      },
+      "mangle": true
+    },
+    "target": "esnext",
```

Comment:
can you change this to `ES2022` please. See https://github.com/digicatapult/veritable-ui/pull/195 for an example of the issue